sklearn/preprocessing/_encoders.py

```diff
@@ -621,6 +622,18 @@ class OrdinalEncoder(_BaseEncoder):
     dtype : number type, default np.float64
         Desired dtype of output.
 
+    handle_unknown : {'error', 'use_encoded_value}', default='error'
+        When set to 'error' an error will be raised in case an unknown
+        categorical feature is present during transform. When set to
+        'use_encoded_value', the encoded value of unknown categories will be
+        set to the value given for the parameter `unknown_value`. In
+        :meth:`inverse_transform`, an unknown category will be denoted as None.
+
+    unknown_value : int, default=None
+        When the parameter handle_unknown is set to 'use_encoded_value', this
+        parameter is required and will set the encoded value of unknown
+        categories.
+
     Attributes
     ----------
     categories_ : list of arrays
```

Comment:
This says that categories_ will correspond with the output of transform. I suspect this is no longer true. Is there a way we can make them line up?

Reply:
Correct, in case of unknown categories there will be outputs of `transform` that are not in `categories_`, namely `unknown_value`. Are you suggesting a code change or just another description here?

Reply:
I'm not sure how to change the code here. But the description, yes.